x-pack/plugins/security_solution/public/cases/containers/api.ts

```diff
@@ -73,6 +81,34 @@ export const getCase = async (
   return convertToCamelCase<CaseResponse, Case>(decodeCaseResponse(response));
 };
 
+export const getSubCase = async (
+  caseId: string,
+  subCaseId: string,
+  includeComments: boolean = true,
+  signal: AbortSignal
+): Promise<Case> => {
+  const [caseResponse, subCaseResponse] = await Promise.all([
+    KibanaServices.get().http.fetch<CaseResponse>(getCaseDetailsUrl(caseId), {
```

Comment:
A note for another PR: Is it possible to avoid the two calls and let the backend handle it?